specification/app/resource-manager/Microsoft.App/stable/2022-03-01/ContainerApps.json

```diff
@@ -634,6 +634,16 @@
             "name"
           ]
         },
+        "ipRestrictionRules": {
```

Comment:
No changes to this file since this is a published version and we cannot make any new changes.